ui/accordion.reel/accordion.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @module accordion
+ * @requires montage/ui/montage
+ * @requires montage/ui/component
+ */
+var Montage = require("montage").Montage,
+    Component = require("montage/ui/component").Component;
+
+/**
+ * @class Accordion
+ * @extends Component
+ */
+exports.Accordion = Montage.create(Component, {
```

Comment:
Yeah, I think so. Just need to know what's the correct way to add example, test and document in info folder, better to have a example